openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml

```
@@ -45,6 +45,11 @@ allOf:
             format: password
             x-multiline: true
             writeOnly: true
+          sftpPrivateKeyPassword:
+            type: string
+            format: password
+            description: SFTP private key password.
```

suggestion: to avoid [noun strings](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
            description: Password of the SFTP private key.
```